doc-content/enterprise-only/employee-rostering/er-rotation-edit-proc.adoc

```diff
@@ -1,14 +1,18 @@
 [id='er-rotation-edit-proc']
 = Viewing and editing the rotation pattern
 
-The rotation pattern contains shifts that are automatically replicated for every period. For details about the rotation pattern, see <<er-rotation-con>>.
+The rotation pattern contains shift templates that are automatically replicated for every period. For details about the rotation pattern, see <<er-rotation-con>>.
 
 You can view the rotation pattern, add, move, and delete shifts.
 
 .Procedure
 . To view and edit the rotation pattern, click the *Rotation* tab. 
 . To view additional days of the rotation pattern, use the horizontal scroll bar at the bottom of the browser window.
-. To add a shift, click an empty area of the table. The application adds a shift, determining the slot and time span automatically from the location of the click.
-. To change the start or end time of a shift, move the mouse pointer to the left or right border of the box representing the shift and drag the border.
-. To delete a shift, hold the *Shift* key on the keyboard and click a shift.
- 
+. To add a shift template, click an empty area of the table. The application adds a shift, determining the slot and time span automatically from the location of the click.
+. To change the start or end time of a shift template, move the mouse pointer to the left or right border of the box representing the shift template and drag the border.
+. To edit a shift template, click the shift template. You can set the following values:
+** *From* and *To* day and time: The exact time and duration of the shift. The day number is counted from the start of the rotation period.
+** *Rotation employee*: The employee assigned to the shift by default. Select *Unassigned* if you do not want to assign any employee to this shift by default; in this case, the employee in the shift is determined by the rostering process in every case without a preferred value.
++
+To save the changes, click *Apply*.
+. To delete a shift template, hold the *Shift* key on the keyboard and click a te shift template. Alternatively, click the shift template and then click *Delete shift template*.
```

Comment:
Just number formatting, because it's a procedure. The question of procedures and numbers vs. actions that are really not sequential is long and complicated. I did the best I could here, with Emily's input.